### PR TITLE
Feature/149 Added withdrawal page

### LINKED ISSUE #149: Add withdrawal page

### PR DESCRIPTION
**Pull request by n-s-mordvinov**

### CODE REVIEW

File: src/pages/Withdraw/index.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    return currencyFormat.format(value).replace(numberFormat.format(value), '').replace(' ', '');
+  };
+
+  const onChangeSum = (e) => {
+    const value = e.currentTarget.value;
+    let result = sum;
+
+    if (value[0] === '.' || value[0] === ',') {
+      result = `0${value}`.replace(',', '.');
+    } else if (/^[\d]*(\.|\,)?[\d]{0,2}$/.test(value)) {
```
Comment (kirillta) [inline src/pages/Withdraw/index.jsx:54]: Better change it on send to disturb a customer less

Comment (sernik73) [reply src/pages/Withdraw/index.jsx:54]: "Intl.NumberFormat" only works through the point

Comment (sernik73) [reply src/pages/Withdraw/index.jsx:54]: I will try to get around

Comment (kirillta) [reply src/pages/Withdraw/index.jsx:54]: I mean, it's confusing if a value changes on input, but we may replace chars right before the value goes to a server

Comment (kirillta) [reply src/pages/Withdraw/index.jsx:54]: In such case it's OK, I think

**Review by kirillta — APPROVED** [review]

**Pull request merged**